ContentTransformer/HtmlStylesRemover.php

```diff
@@ -23,33 +23,27 @@ public function __construct(string $classNamePrefix = 'removable-style-')
         $this->classNamePrefix = $classNamePrefix;
     }
 
-    public function canTransform(ContentTransformContext $contentTransformContext = null): bool
+    public function canTransform(ContentTransformContext $contentTransformContext): bool
     {
-        if (!$contentTransformContext) {
+        if (!$contentTransformContext->getDataFieldType() instanceof WysiwygFieldType) {
             return false;
         }
 
-        foreach ($contentTransformContext->get() as $dataFieldType) {
-            if (!$dataFieldType instanceof WysiwygFieldType) {
-                return false;
-            }
-
-            return true;
-        }
+        return true;
     }
 
-    public function transform(string $input): string
+    public function transform(ContentTransformContext $contentTransformContext): string
     {
-        $this->input = $input;
-        $this->doc = $this->initDocument($input);
+        $this->input = $contentTransformContext->getData();
```

Comment:
Why are we keeping state in this class? 
Can't we pass ->getData() as context to the other functions?